src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -692,7 +692,12 @@
   product(uint, GCCardSizeInBytes, 512,                                     \
           "Card table entry size (in bytes) for card based collectors")     \
           range(128, NOT_LP64(512) LP64_ONLY(1024))                         \
-          constraint(GCCardSizeInBytesConstraintFunc,AtParse)
+          constraint(GCCardSizeInBytesConstraintFunc,AtParse)               \
+                                                                            \
+  /* SapMachine 2023-09-25 */                                               \
+  product(bool, UseParallelLargeArrayScanning, false,                       \
```

Comment:
Should it be EXPERIMENTAL? I guess we will remove the flag again once we have the final version backported.

Reply:
I didn't bother but probably it's better to make it experimental.

Reply:
Done :)